src/components/Api.js

```diff
@@ -0,0 +1,93 @@
+export class Api {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._authToken = headers.authorization;
+    this._headers = headers;
+  }
+
+  /*User routes*/
+  //Get the current user's info
+  getUserInfo() {
+    return fetch(`${this._baseUrl}/users/me`, { headers: this._headers })
+      .then((res) => (res.ok ? res.json() : Promise.reject(`Error: ${res.status}`)))
+      .catch(console.error);
```

Comment:
It’s better not to use `catch` in `Api` because you need to `catch` possible errors at the very **end** of any request and this end is in `index.js`  after all blocks `then`. And in real projects blocks `catch` usually open error popups and so on.

Please, remove it from `Api` and add it in `index.js`.